Provide a way to override enabling wasm from CMake
by setting -DENABLE_WEBASSEMBLY=OFF for RISCV64

Upstream-Status: Submitted [https://bugs.webkit.org/show_bug.cgi?id=254553]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

--- a/Source/WTF/wtf/PlatformEnable.h
+++ b/Source/WTF/wtf/PlatformEnable.h
@@ -613,13 +613,14 @@
 #endif
 
 #if CPU(RISCV64)
-#undef ENABLE_WEBASSEMBLY
+#if !defined(ENABLE_WEBASSEMBLY)
 #define ENABLE_WEBASSEMBLY 1
 #undef ENABLE_WEBASSEMBLY_B3JIT
 #define ENABLE_WEBASSEMBLY_B3JIT 0
 #undef ENABLE_WEBASSEMBLY_BBQJIT
 #define ENABLE_WEBASSEMBLY_BBQJIT 0
 #endif
+#endif
 
 #if !defined(ENABLE_C_LOOP)
 #if ENABLE(JIT) || CPU(X86_64) || CPU(ARM64)
